githooks/pre-commit

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+# To use this githook copy this file to .git/hooks (in repo root dir)
+# The file permissions may need changing using ``chmod +x .git/hooks/pre-commit``
+
+
+echo Running formatter...
+TEST_RESULTS=$(tox -e format)
+RETURN_VALUE=$?
+if [ $RETURN_VALUE != 0 ]; then
+  echo "$TEST_RESULTS"
+  exit 1
+fi
+echo Finished
+exit 0
```

Comment:
Letting this go in but should get removed post https://github.com/ssec-jhu/base-template/pull/136

Reply:
Doh, only just noticed that this is totally new and pre-commit and not the pre-push I thought it was.